src/openvic-simulation/country/CountryInstance.cpp

```diff
@@ -10,6 +10,8 @@
 #include "openvic-simulation/politics/Ideology.hpp"
 #include "openvic-simulation/research/Invention.hpp"
 #include "openvic-simulation/research/Technology.hpp"
+#include "openvic-simulation/types/Sliders.hpp"
```

Comment:
Following the code style of this repo, includes are sorted into groups (usually `<STL headers>`, maybe `<openvic-dataloader headers>`, and then `"openvic-simulation headers"`) and then alphabetically ordered within those groups. In this case, the `Sliders.hpp` include should go after the `fixed_point/FixedPoint.hpp` include